tests/sys/Makefile

```diff
@@ -42,7 +42,7 @@ TESTS_SUBDIRS+=		vmm
 # during cleanup. They start and stop a new auditd for every test CASE which
 # takes a really long time on QEMU. In the latest jenkins run these tests took
 # 9.5 hours to run! Skip them until we can run them at a reasonable speed.
-.if ${MK_AUDIT} != "no" && ${MK_CHERI} == "no"
+.if ${MK_AUDIT} != "no" && ${MACHINE_CPU:Mcheri}
```

Comment:
I think this needs to be `!${MACHINE_CPU:Mcheri}`?

Reply:
Hmm I feel like I fixed that upstream and we can drop this local diff.

Reply:
I think https://github.com/CTSRD-CHERI/cheribsd/commit/df093aa9463b2121d8307fb91c4ba7cf17f4ea64 and https://github.com/CTSRD-CHERI/cheribsd/commit/869cc06480b75b4caea0d049e0cf7f82bb5aeed1 may have improved that situation.